tests/test_addsubdir.py

```diff
@@ -0,0 +1,61 @@
+""" 
+test_addsubdir.py --- test if build_component.py can copy files to subdirectories 
+of the build target dir.
+
+The scripts can call build_component.py directly not initialize.py, so you should 
+execute initialize.py before running it.
+
+""" 
+import os 
+import subprocess
+def main():
+	# set component directory
+	component_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+	# change directory to component directory ../../common/
+	os.chdir(component_dir)
+	#subprocess.call(["python", "initialize.py"])
```

Comment:
Should we remove this commented like of code?  Whitespace between comments and code would also help readability.